frontend/src/Event/MeetEventDisplay.jsx

```diff
@@ -14,13 +14,19 @@ import {
 import MyButton from "../components/FormElements/MyButton.jsx";
 import { Stack, Box } from "@mui/material";
 import { useParams, useNavigate, useLocation } from "react-router-dom";
+import { Build as BuildIcon } from "@mui/icons-material";
```

Comment:
To clean up the imports, could you, please, merge all the icon imports into a single line.

Reply:
Done, I moved BuildIcon along the other icons.